skxray/core/roi.py

```diff
@@ -430,11 +432,23 @@ def circular_average(image, calibrated_center, threshold=0, nx=100,
     ring_averages : array
         Radial average of the image. shape is (nx, ).
     """
-    radial_val = utils.radial_grid(calibrated_center, image.shape,
-                                   pixel_size)
+    radial_val = utils.radial_grid(calibrated_center, image.shape, pixel_size)
+
+    if min_x is None:
+        min_x = np.min(radial_val)
+    else:
+        min_x = min_x * pixel_size[0]
+    if max_x is None:
+        max_x = np.max(radial_val)
+    else:
+        max_x = max_x * pixel_size[0]
+    if nx is None:
```

Comment:
This check should probably be deleted or the behavior of passing in `None` causes 1 pixel bins to be documented.